umm-spec-lib/test/cmr/umm_spec/test/xml_to_umm_mappings/iso19115_2/spatial.clj

```diff
@@ -27,3 +28,14 @@
        "genericresolutions" :GenericResolutions,
        "griddedresolutions" :GriddedResolutions,
        "griddedrangeresolutions" :GriddedRangeResolutions})))
+
+(deftest test-coordinate-system-parsing
+  "Testing the parsing of the spatial coordinate system. The values of CARTESIAN and GEODETIC
+   are tested with other tests. Data that contains EPSG will be tested here."
+
+  (testing "test the parsing of the spatial coordinate system.")
```

Comment:
Need to include the `let` within the `testing` sexp and the text above could be a comment